client/app/account/billing/history/billing-history.controller.js

```diff
@@ -202,27 +214,45 @@ angular.module("Billing.controllers").controller("Billing.controllers.History",
             });
     }
 
-    (function init (_self) {
+    this.$onInit = function () {
+        this.isLoading = true;
         BillingUser.isVATNeeded().then((result) => {
-            _self.isVATNeeded = result;
+            this.isVATNeeded = result;
         });
+        this.canSetInvoiceByMail = false;
 
         loadDefaultPaymentMethod();
 
-        BillingDebtAccount.getDebtAccount()
-            .then((debtAccount) => {
-                _self.debtAccount = debtAccount;
-                if (_self.debtAccount.active) {
-                    _self.colSpan = COL_SPAN_DEBT_ACCOUNT;
+        return $q.all([
+            BillingDebtAccount.getDebtAccount()
+                .then((debtAccount) => {
+                    this.debtAccount = debtAccount;
+                    if (this.debtAccount.active) {
+                        this.colSpan = COL_SPAN_DEBT_ACCOUNT;
+                    }
+                })
+                .catch((err) => {
+                    if (err.status === 404) {
+                        return null;
+                    }
+                    return $scope.setMessage($scope.tr("billingError"), {
+                        alertType: "ERROR"
+                    });
+                }),
+            OvhApiMe.v6().get().$promise.then((user) => {
+                if (user.country === "FR") {
+                    return OvhApiMe.Billing().InvoicesByPostalMail().v6().get().$promise
+                        .then((result) => {
+                            this.canSetInvoiceByPostalMail = true;
+                            this.invoicesByPostalMail = result.data;
+                            this.tmpInvoicesChoice = angular.copy(this.invoicesByPostalMail);
+                        });
                 }
+
+                return user;
```

Comment:
you shouldn't return anything here (or you should also return user in the branching "if" for consistency. But since return value is not used ...)